eclair-node/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -283,6 +283,9 @@ trait Service extends ExtraDirectives with Logging {
                         path("usablebalances") {
                           complete(eclairApi.usableBalances())
                         } ~
+                        path("onchainbalances") {
```

Comment:
Yes, that bothered me too :smile: 
I think that since for off-chain we have `usablebalances` we need to align this one and the on-chain one. I just don't know if plural or singular is better, what do you prefer?
It's probably a good occasion to rename `usablebalance(s)` to `offchainbalance(s)`? Or something else?

Reply:
The plural for LN balances is for `send`/`receive`, so it doesn't bother me to have `usablebalances` and `onchainbalance`.

Reply:
Great in that case I'll go with the simplest change of renaming the JSON API `onchainbalance`

Reply:
Done in https://github.com/ACINQ/eclair/pull/1461/commits/80b23340a99b886b6df1df79551b4c2b33ab093a